src/webui/www/private/scripts/mocha-init.js

```diff
@@ -120,9 +120,9 @@ const initializeWindows = function() {
         const id = 'downloadPage';
         let contentUrl = 'download.html';
         if (urls && (urls.length > 0)) {
-            contentUrl += ('?urls=' + urls.map(function(url) {
+            contentUrl += ('?urls=' + encodeURIComponent(urls.map(function(url) {
```

Comment:
@brvphoenix 
I'm looking into issue #14695 and found this was the culprit. Could you explain why this change?

Reply:
https://github.com/qbittorrent/qBittorrent/pull/14570#discussion_r599322750
Same as here.

Reply:
>If the paths or rules contain '|'....

That condition applies here too? IIRC this code path is targeted for webui only, users should not be expecting on some specific interface. Or you have better suggestions to solve #14695 than reverting this?

Reply:
> Or you have better suggestions to solve #14695 than reverting this?

I suspect it is caused by the browser transmitting encoded urls to the qbittorrent.

Reply:
@ Chocobo1 
It has been fixed in #14703.